root/search/components/EventResults.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+import uniq from 'lodash/uniq';
+
+import {withCatalystContext} from '../../context';
+import {l} from '../../static/scripts/common/i18n';
+import {l_attributes} from '../../static/scripts/common/i18n/attributes';
+import commaOnlyList from '../../static/scripts/common/i18n/commaOnlyList';
+import PaginatedResults from '../../components/PaginatedResults';
+import DescriptiveLink from '../../static/scripts/common/components/DescriptiveLink';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import formatDatePeriod from '../../static/scripts/common/utility/formatDatePeriod';
+import loopParity from '../../utility/loopParity';
+import type {ResultsPropsT} from '../types';
+
+import ResultsLayout from './ResultsLayout';
+
+function buildResult(result, index) {
+  const event = result.entity;
+  const score = result.score;
+
+  return (
+    <tr className={loopParity(index)} key={event.id}>
+      <td>{score}</td>
+      <td>
+        <DescriptiveLink entity={event} />
+      </td>
+      <td>{event.typeName ? l_attributes(event.typeName) : null}</td>
```

Comment:
Context is missing for localization, should be:
```js
lp_attributes(event.typeName, 'event_type')
```
I just found the same bug for area results and artist results components.

Reply:
Thanks for the fix. :)